core/tests/puppeteer-acceptance-tests/puppeteer-testing-utilities/user-factory.js

```diff
@@ -166,6 +168,20 @@ let createNewQuestionAdmin = async function(username, email) {
   activeUsers.push(questionAdmin);
   return questionAdmin;
 };
+/**
+ * Function to create a user with the question admin role.
+ * @param {string} username - the username of the question admin.
+ * @param {string} email - the email of the user.
+ * @returns the instance of the question admin.
+ */
+
+let createNewExplorationAdmin = async function(username, email) {
+  const explorationAdmin = new e2eExplorationAdmin();
+  await explorationAdmin.openBrowser();
+  await explorationAdmin.signUpNewUser(username, email);
+  activeUsers.push(explorationAdmin);
+  return explorationAdmin;
```

Comment:
We don't have a role called "exploration admin" in the codebase. Maybe you mean "exploration creator" or "exploration editor"?

Reply:
Made changes as per suggestion.